### PR TITLE
Error with react-google-maps/api 

### LINKED ISSUE #20: Set up Maps
## User story

User should be able to see map an choose Start with it. 

## Notes / Links / Further Description
https://www.npmjs.com/package/google-map-react

## ToDos

-[ ] read docs
-[ ] set in dark mode
-[ ] styling

### PR DESCRIPTION
**Pull request by Phil9219**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,6 +71,7 @@ typings/
 # dotenv environment variables file
 .env
 .env.test
+.env.local
```
Comment (lmachens) [inline .gitignore:74]: What's the reason for this change?

File: client/package.json [hunk 2]
```diff
@@ -14,13 +16,16 @@
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
     "babel-loader": "^8.1.0",
+    "date-fns": "^2.16.1",
     "eslint": "^7.14.0",
     "eslint-plugin-react": "^7.21.5",
     "react": "^17.0.1",
     "react-datepicker": "^3.3.0",
     "react-dom": "^17.0.1",
+    "react-google-maps": "^9.4.5",
```
Comment (lmachens) [inline client/package.json]: Is this used? Consider removing it if it's unused.

File: client/src/App.js [hunk 3]
```diff
@@ -3,16 +3,18 @@ import AppHeader from "./components/AppHeader/AppHeader";
...
 import Button from "./components/Button/Button";
 import React from "react";
+import Map from "./components/Map/Map";
 
 function App() {
   return (
     <div className="App">
-      <GlobalStyle />
+      <Map />
+      {/* <GlobalStyle />
```
Comment (lmachens) [inline client/src/App.js]: I know that this is a draft request, but consider removing these lines instead of comment them out to avoid having unused code.

File: client/src/components/Map/Map.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+            onCloseClick={() => {
+              setSelected(null);
+            }}
+          >
+            <div>
+              <h2>Run Start</h2>
+              <p>Be Ready At {formatRelative(selected.time, new Date())}</p>
+            </div>
+          </InfoWindow>
+        ) : null}
```
Comment (lmachens) [inline client/src/components/Map/Map.js]: If you don't have an `else` case, consider using`&&`.

```suggestion
        {selected && (
          <InfoWindow
            position={{ lat: selected.lat, lng: selected.lng }}
            onCloseClick={() => {
              setSelected(null);
            }}
          >
            <div>
              <h2>Run Start</h2>
              <p>Be Ready At {formatRelative(selected.time, new Date())}</p>
            </div>
          </InfoWindow>
        )}
```

Comment (lmachens) [reply client/src/components/Map/Map.js]: This would be a great component 
```suggestion
        {selected && <RunInfoWindow
            position={{ lat: selected.lat, lng: selected.lng }}
            onCloseClick={() => {
              setSelected(null);
            }}
            time={selected.time}
          />}
```

File: client/src/components/Map/Map.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          () => null
+        );
+      }}
+    >
+      🧭
+    </button>
+  );
+}
+
+function Search(panTo) {
```
Comment (lmachens) [inline client/src/components/Map/Map.js]:
```suggestion
function Search({ panTo }) {
```

File: client/src/components/Map/Map.js [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+              <p>Be Ready At {formatRelative(selected.time, new Date())}</p>
+            </div>
+          </InfoWindow>
+        ) : null}
+      </GoogleMap>
+    </div>
+  );
+}
+
+function Locate(panTo) {
```
Comment (lmachens) [inline client/src/components/Map/Map.js]: `panTo` is not working like expected.

```suggestion
function Locate({ panTo }) {
```

**Review by lmachens — COMMENTED** [review]
Good job 👍

File: client/package.json (hunk 2 above)
Comment (Phil9219) [reply client/package.json]: I uninstalled it. Thanks

File: client/src/components/Map/Map.js (hunk 5 above)
Comment (Phil9219) [reply client/src/components/Map/Map.js]: Got it. Thanks
